src/Kros.ApplicationInsights.Extensions/FilterSyntheticRequestsProcessor.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.Extensibility;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kros.ApplicationInsights.Extensions
+{
+    /// <summary>
+    /// Telemetry Processor to filter out synthetic requests(bots, web search...).
+    /// </summary>
+    /// <seealso cref="Microsoft.ApplicationInsights.Extensibility.ITelemetryInitializer" />
+    class FilterSyntheticRequestsProcessor : ITelemetryProcessor
```

Comment:
Use explicit visibility modifier. Probabli `internal`.

Reply:
Write unit tests for this class.

Reply:
Added unit test and modifier